tests/ui/components/questions/questionsubmissioncomponent.js

```diff
@@ -0,0 +1,36 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import QuestionSubmissionComponent from '../../../../src/ui/components/questions/questionsubmissioncomponent';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SearchStates from '../../../../src/core/storage/searchstates';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig;
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  defaultConfig = {
+    container: '#test-component',
+    entityId: 1234
+  };
+});
+
+describe('QASubmission component', () => {
+  it('listens to updates to VERTICAL_RESULTS in global storage', () => {
+    const storage = COMPONENT_MANAGER.core.storage;
+    const component = COMPONENT_MANAGER.create(QuestionSubmissionComponent.type, defaultConfig);
+    const wrapper = mount(component);
+    expect(wrapper.find('.yxt-QuestionSubmission')).toHaveLength(0);
```

Comment:
nit: Assuming these tests are testing whether or not the component is visible, what are your thoughts on an explanatory variable so that the code reads: `expect(componentIsVisible).toBeFalsy()` and `expect(componentIsVisible).toBeTruthy()`? The variable would be `const componentIsVisible = wrapper.find('.yxt-QuestionSubmission').length >= 1`

Reply:
we can use .exists() here, would that be better? (updated the code with this)

Reply:
Ya, that looks good!